modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -697,6 +697,11 @@ public class UpgradeCatchAllCheck implements ConfigurationBeanDeclaration {
 		contact.setSuffixId();
 	}
 
+	public void method(
+		CustomFacetPortletPreferences customFacetPortletPreferences) {
+		customFacetPortletPreferences.getFederatedSearchKeyOptional();
```

Comment:
@regiCesar21 Hi, after breaking a line on the method declaration, we leave a blank line before starting the code. We should only start right after if the declaration happened in line.